docs_user/modules/proc_stopping-openstack-services.adoc

```diff
@@ -25,26 +25,21 @@ Note that you should not stop the infrastructure management services yet, such a
 
 .Prerequisites
 
-You can stop {OpenStackShort} services at any moment, but you might leave your environment in an undesired state. However, you should confirm that there are no long running operations that require other services.
-
-Ensure that there are no ongoing instance live migrations, volume migrations (online or offline), volume creation, backup restore, attaching, detaching,
+* You can stop {OpenStackShort} services at any moment, but you might leave your environment in an undesired state. However, you should confirm that there are no long running operations that require other services.
```

Comment:
Do we really want to document this content? If there's a risk of an "undesirable state" I wonder if this is wise?

- `moment` seems awkward to me. Maybe `time` instead? 
- Per ISG:
````
'Do not use [etc.]. Use "and so on" when you list a clear sequence of
elements, such as "1, 2, 3, and so on"....'
````
If you do decide to document this, maybe change it to something like this?
````
Before you stop {OpenStackShort} services, first confirm that there no long 
running operations that require these services.
WARNING: Avoid stopping {OpenStackShort} services. Doing so can leave
your environment in an undesired state.
````